test/components/test-harness-component.html

```diff
@@ -3,17 +3,52 @@
 <md-button ng-click="$ctrl.navigate('/routeb');" type="button" class="md-primary md-raised">Route B</md-button>
 <md-button ng-click="$ctrl.navigate('/routec');" type="button" class="md-primary md-raised">Route C</md-button>
 <h1>Create Identity</h1>
-<form name="$ctrl.identityForm">
+<form name="$ctrl.identityForm" class="br-flex-vertical-children br-flex-stretch-children-width">
   <br-input br-model="$ctrl.sysSlug"
     br-options="{
       name: 'slug',
-      label: 'Short Name'
-    }">
+      label: 'Email Address',
+      theme: 'material',
+      type: 'email',
+      showValidation: true
```

Comment:
in response to your difficulty with errors.  I believe with the material theme, showValidation should be removed.  I noted that that seems to be the only difference between implementation here and in bedrock-angular-authn-password.